.semgrep/custom-rules.yaml

```diff
@@ -47,7 +47,7 @@ rules:
   languages:
     - python
   severity: ERROR
-  message: Possible path traversal or insecure directory and file permissions through os.mkdir(). Use securedrop_export.utils.safe_mkdir instead.
+  message: Possible path traversal or insecure directory and file permissions through os.mkdir(). Use securedrop_export.directory_util.safe_mkdir instead.
```

Comment:
The reason I called it this way, which may or may not be valid, is that it's not a class - it's a series of functions (utilities) that are imported individually to classes that need them, but are unfortunately used in multiple places so I didn't want to make them class methods since that wouldn't be DRY.  Up to you if you want it changed or not.

Reply:
I get that. I'd day that the `logging` module defines functions, it is not called `logging_util` for that. Nor is `os` called `os_utils`.

I see a convention, but I think it is not a two-way convention: when defining a class (e.g. `Dog`) it is nice for the module to be called `dog` (file: `dog.py`), but the fact that a module is called `cat` doesn't imply that a `Cat` class must be defined in it. 

I was suggesting renaming the file _only_, not changing anything inside.

Reply:
Addressed in https://github.com/freedomofpress/securedrop-export/pull/105/commits/ed6ea1e6147f6e764eeeec679c5c246e5570bdb7